site2/docs/admin-api-topics.md

```diff
@@ -538,6 +538,31 @@ admin.topics().getMessageById(topic, ledgerId, entryId);
 
 <!--END_DOCUSAURUS_CODE_TABS-->
 
+### Get message ID 
+
+You can get message id published at or just after the given datetime.
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--pulsar-admin-->
+```shell
+$ ./bin/pulsar-admin topics get-message-id \
```

Comment:
```suggestion
./bin/pulsar-admin topics get-message-id \
```

Remove $ so that users can copy the command directly and conveniently.

Reply:
There are $ in all the other shell demos. You sure we should do that? Either is fine by me.

Reply:
@Anonymitaet please take a look Jason's comment

Reply:
@Jason918 we're removing the $ (though there are still some $ exist), please keep consistent

Reply:
@Anonymitaet  OK, I have commited this change. Please take a look.